lib/card.rb

```diff
@@ -0,0 +1,8 @@
+class Card
+  attr_reader :question, :answer, :category
```

Comment:
Now that we have a better understanding of `attr_reader`'s, let's push it a bit further and ask: do we want this to be public or private?

https://www.peterdebelak.com/blog/private-attr-readers/

Reply:
do we just add multiple attr_reader's if we want one public and one private

Reply:
We can do that! In this case, since we are trying to expose them to other classes, it's okay that they are public. That may not be the case all the time, though, so it'll be good to keep this in mind in development. :-)